frontend/src/assets/scss/tabs.scss

```diff
@@ -3,22 +3,32 @@
     @apply m-0 border-none #{!important};
   }
   &__active-bar {
-    @apply bg-primary-500 #{!important};
+    @apply bg-transparent #{!important};
   }
```

Comment:
Hey, this file is still from the old approach.
Gasper has created new components for the new design Nuno has been building inside the ui-kit folder.

If you check `ui-kit/tabs/Tabs.vue`, you can see the tabs tab I believe should be used

Reply:
Can you also revert the changes on this file? I believe this will update other tabs in the app which is not desirable

Reply:
I reverted this and used the one in the ui-kit. However, I notice the one in the ui-kit doesn't match the designs. What should be done in the case?